vignettes/web_only/BioCro-II_Paper--Section-1.1-example.Rmd

```diff
@@ -222,9 +223,9 @@ xyplot(Leaf ~ time / 3600, data = result, xlim = c(0, 24), xlab = "time (in hour
 ## Second modification: Changing the time specification from seconds to days
 
 The usual assumption in BioCro is that the `time` quantity represents
-the time, in days, from the beginning of the year, not the number of
-seconds that have elapsed from the start of the simulation.  (More
-precisely, the integral part of the `time` value should be the day of
+the time, in hours, that has elapsed from the beginning of the year,
+not the number of seconds that have elapsed from the start of the simulation.
+(More precisely, the integral part of the `time` value should be the day of
```

Comment:
Hasn't this changed to the integral part is hours? I don't know whether now it starts at 1 or 0.